epersongroups.md

```diff
@@ -0,0 +1,390 @@
+    },
+    "epersons": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+    },
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc"
+    }
+  }
+}
+```
+
+## Create new EPerson Group
+
+**POST /api/eperson/groups**
+
+To create a new EPerson Group, perform a post with the JSON below to the epersons endpoint when logged in as admin.
+
+```json
+{
+  "metadata": {
+      "dc.title": [
+        {
+          "value": "New Group 1",
+          "language": null,
+          "authority": "",
+          "confidence": -1
+        }
+      ]
+  }
+}
+```
+
+## Parent Groups of a single parent EPerson Group
+
+### Get Parent Groups of a single parent EPerson Group
+
+**GET /api/eperson/groups/<:uuid>/parentgroups**
+
+Retrieves direct parent groups of the given group.
+Requesting indirect parent groups is not supported
+
+Sample:
+* GET /rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/parentgroups
+
+```json
+{
+    "_embedded": {
+      "groups": [
+        {
+          "id": "3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42",
+          "uuid": "3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42",
+          "name": "Parent group",
+          "handle": null,
+          "metadata": {
+              "dc.title": [
+                {
+                  "value": "Parent group",
+                  "language": null,
+                  "authority": "",
+                  "confidence": -1,
+                  "place": 0
+                }
+              ]
+          },
+          "permanent": true,
+          "type": "group",
+          "_links": {
+            "subgroups": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/groups"
+            },
+            "parentgroups": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/parentgroups"
+            },
+            "epersons": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/epersons"
+            },
+            "self": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42"
+            }
+          }
+        }
+      ]
+    },
+    "_links": {
+      "self": {
+        "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/groups"
+      }
+    },
+    "page": {
+      "number": 0,
+      "size": 10,
+      "totalPages": 1,
+      "totalElements": 10
+    }
+}
+```
+
+## Sub Groups in a single parent EPerson Group
+
+### Get Sub Groups in a single parent EPerson Group
+
+**GET /api/eperson/groups/<:uuid>/subgroups**
```

Comment:
this should become /groups as in the current implementation

Reply:
I actually used subgroups to make sure it's clear which groups it returns.
This ensures it's in line with https://github.com/DSpace/Rest7Contract/blob/master/communities.md#subcommunities

The old implementation (which didn't even have a REST contract) shouldn't be a reason to use harder to read URLs

Reply:
@benbosman : I'm OK with that approach. I was working under the assumption that we were attempting to align this contract with the current implementation. However, if you'd rather *change* the current implementation, I'm OK with renaming this subresource to `subgroups`. Thanks for clarifying your goal here.